pkg/daemon/ceph/osd/volume.go

```diff
@@ -220,17 +221,18 @@ func (a *OsdAgent) initializeBlockPVC(context *clusterd.Context, devices *Device
 	// we need to return the block if raw mode is used and the lv if lvm mode
 	baseCommand := "stdbuf"
 	var baseArgs []string
+	storeFlag := fmt.Sprintf("--%s", a.storeConfig.StoreType)
```

Comment:
How about a helper method for creating this flag? I see this same formatting here, line 516, and in osd/spec.go that could be refactored.